docs/_includes/themes/zeppelin/_navigation.html

```diff
@@ -66,6 +66,7 @@
                 <li><a href="{{BASE_PATH}}/interpreter/scalding.html">Scalding</a></li>
                 <li><a href="{{BASE_PATH}}/interpreter/shell.html">Shell</a></li>
                 <li><a href="{{BASE_PATH}}/interpreter/spark.html">Spark</a></li>
+                <li><a href="{{BASE_PATH}}/interpreter/bigquery.html">BigQuery</a></li>
```

Comment:
As you can see, the lists are in alphabetical order. You need to put `bigquery.html` between `alluxio.html` and `cassandra.html`.